src/Sylius/Bundle/CoreBundle/Resources/config/services/integrations/payum/stripe.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
+    <services>
+        <defaults public="true" />
+
+        <service id="sylius.payment_request.payum.command_provider.stripe" class="Sylius\Bundle\CoreBundle\PaymentRequest\CommandProvider\PaymentRequestTypeCommandProvider">
+            <argument type="tagged_locator" tag="sylius.payment_request.payum.command_provider.stripe" index-by="type" />
+            <tag name="sylius.payment_request.command_provider" factory="stripe" />
```

Comment:
stripe_checkout is not what this gateway is using, it is using stripe charge API, stripe checkout is the Stripe external portal.

Reply:
That begs a different question then, should we cover processing gateways that cannot be configured via admin? The only types available by default are `paypal_checkout_express`, `stripe_checkout` and `offline`.

Reply:
Never-mind, it's historical, the `payum/stripe` library called the gateway `stripe_checkout` before the Stripe Checkout Session exists. Since `stripe_checkout_session` and `stripe_checkout` have a different name we can keep `stripe_checkout` as the factory name inside this tag, but it is confusing (only if you already know how Stripe is working right now I guess).